app/views/topical_events/show.html.erb

```diff
@@ -155,6 +155,15 @@
                   <a href="http://www.warmemorials.org"><%= t("topical_event.lead_organisations.war_memorials_trust") %></a>
                 </li>
               <% end %>
+
+              <% if @topical_event.slug == '2022-events-platinum-jubilee-commonwealth-games-festival-uk' %>
+                <li class="non-govuk birmingham_2022_commonwealth_games">
+                  <a href=""><%= t("topical_event.lead_organisations.birmingham_2022_commonwealth_games") %></a>
```

Comment:
Is this the bit RE "Currently the Birmingham org is linked via whitehall due to being listed as a lead org for the event"? I don't understand how Whitehall populates this `href` (if I've understood correctly).

The rest of the PR looks good to me 👍

Reply:
Whitehall has them listed as a lead org but that doesn't allow them to display a logo, only "live" government departments can do that, hence the need to hard code. I will go and adjust so they're listed as involved but no longer lead to prevent a duplicate link.

Reply:
Ah, but I've failed to hard code the link, I'll fix that now. Thanks!